assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -517,6 +517,10 @@ $card-preview-zoom-bottom-offset:       6rem;
     }
 }
 
+.cart-item-discount, .cart-item-value.price--discounted {
+    color: stencilColor("color-greyLight");
```

Comment:
hi @richapatel1510 👋 I might be missing something, but are we considering the fact that the theme has three style variations? Will this colour still match  rest of the theme if the user switches to a different variation? For example:
<img width="950" height="813" alt="image" src="https://github.com/user-attachments/assets/7c0e5fc7-955e-481b-82f4-0c8c1502aef8" />

If not, maybe we can take an example from `cart-total-label`. Here are  [scss](https://github.com/bigcommerce/cornerstone/blob/master/assets/scss/components/stencil/cart/_cart.scss#L368) settings and here is a [config](https://github.com/bigcommerce/cornerstone/blob/master/config.json#L142) where it applies across all versions.